CSS/syntax_test_css.css

```diff
@@ -2082,9 +2082,9 @@
 /*              ^ meta.property-list.css - meta.property-name - meta.property-value */
 /*               ^^^^^^^^^^ meta.property-list.css meta.property-value.css */
 /*                         ^ meta.property-list.css - meta.property-name - meta.property-value */
-/*  ^^^^^^^^^^^^ - entity - support */
+/*  ^^^^^^^^^^^^ support.type.property-name.css */
 /*              ^ punctuation.separator.key-value.css */
-/*                ^^^^^^^^^ constant.other.css - support */
+/*                ^^^^^^^^^ support.constant.property-value.css */
```

Comment:
Can you also add an arbitrary/custom property match without any part being known or do we already have that? From the diff, I didn't get that impression at least.

Reply:
I'd say this is exactly this, isn't it?

The `-font-family` starts with a hyphon, which is invalid and would have therefore not been highlight before. This test indicated syntax doesn't care about it anymore. Or is it the leading hyphon, which is misleading?

The value `sans-none` would be scoped `string.unquoted` when used in a valid `font-family` property. In any other known or unknown property it is treated as normal constant.

Reply:
Then it's fine. I just wanted to make sure I'm understanding the test correctly.